addons/calendar/models/calendar_event.py

```diff
@@ -535,6 +535,9 @@ def get_interval(self, interval, tz=None):
             dummy, format_time = self._get_date_formats()
             result = tools.ustr(date.strftime(format_time + " %Z"))
 
+        elif interval == 'stime':
+            result = babel.dates.format_time(date, format='HH:mm', locale=get_lang(self.env).code)
```

Comment:
I think `format='short'` is the correct format.
for us, it will be displayed as HH:mm, but for US, it will handle AM/PM